code/modules/mob/living/simple_animal/abnormality/he/pisc_mermaid.dm

```diff
@@ -59,6 +59,37 @@
 	var/obj/item/clothing/head/unrequited_crown/crown
 	var/mob/living/carbon/human/love_target
 
+	attack_action_types = list(
+		/datum/action/innate/change_icon_merm
```

Comment:
preferably if you are doing multi-line lists, always have an object end with a comma. It makes future additions look better on github
```suggestion
		/datum/action/innate/change_icon_merm,
```